plugins/woocommerce/client/admin/config/development.json

```diff
@@ -15,6 +15,7 @@
 		"minified-js": true,
 		"mobile-app-banner": true,
 		"navigation": true,
+		"new-product-management-experience": true,
```

Comment:
Having this as `true` is fine for now I think, given we have the MVP button anyway.
Once the MVP button is gone, we might want to turn this back to false and have everyone enabled it themselves.

Reply:
We usually leave the feature flags turned on in dev since it's a work in progress and the dev environment is exactly to work on those things. I'm not opposed to turning the flag off, but if we follow that pattern, I do think that we should turn off all of them in dev.